registry.go

```diff
@@ -75,20 +75,22 @@ func Credential(store Store) func(context.Context, string) (auth.Credential, err
 	}
 }
 
-// The Docker CLI expects that the 'docker.io' credential
-// will be added under the key "https://index.docker.io/v1/"
 func mapStoreRegistryName(registry string) string {
+	// The Docker CLI expects that the 'docker.io' credential
+	// will be added under the key "https://index.docker.io/v1/"
+	// See: https://github.com/moby/moby/blob/master/registry/config.go#L25-L48
```

Comment:
Never use a branch name (e.g. `master`) in the link. Use tag or commit digest so that the link won't expire.

Reply:
Updated the link to: `https://github.com/moby/moby/blob/v24.0.0-beta.2/registry/config.go#L25-L48`